docs/testing.md

```diff
@@ -99,6 +116,10 @@ In the newly created app:
 * tap the `'+'` icon on the app
 * verify that the IDE pauses at the breakpoint, and that the `Variables` pane has
   the right value for `_counter`
+* open the inspector
+* change the display by clicking `Render Tree` and the refresh button
+* open the performance view
+* verify that the memory view is active and the performance view is not
```

Comment:
prefix with: `(in Android Studio):`

Currently the memory view only displays in Android Studio